runtime/integration-tests/src/cases/lp/utils.rs

```diff
@@ -86,15 +86,14 @@ pub fn pool_c_tranche_1_id<T: Runtime>() -> TrancheId {
 }
 
 pub fn verify_outbound_failure_on_lp<T: Runtime>(to: H160) {
-	let (_tx, status, receipt) = pallet_ethereum::Pending::<T>::get()
+	let (_tx, status, _receipt) = pallet_ethereum::Pending::<T>::get()
 		.last()
 		.expect("Queue triggered evm tx.")
 		.clone();
 
 	// The sender is the sender account on the gateway
 	assert_eq!(T::Sender::get().h160(), status.from);
 	assert_eq!(status.to.unwrap().0, to.0);
-	assert!(!receipt_ok(receipt));
```

Comment:
Note - given that the message processor implementation of the LP gateway is transaction, this check is no longer required.